### PR TITLE
feat(components-native): Migrate ThumbnailList

### PR DESCRIPTION
**Pull request by MichaelParadis**
### Added

- components-native: migrate `ThumbnailList`

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

On Jobber Mobile using  `test-thumbnail-list` verify thumbnail list works as expected.

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/ThumbnailList/ThumbnailList.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import { styles } from "./ThumbnailList.style";
+import { FormatFile } from "../FormatFile";
+import { CreateThumbnail, File } from "../FormatFile/types";
+
+export enum RowCount {
+  TwoRows = 2,
+  ThreeRows = 3,
+}
+
+export interface ThumbnailListProps {
```
Comment (taylorvnoj) [inline packages/components-native/src/ThumbnailList/ThumbnailList.tsx:13]: Should we put the above `enum` and this `interface` in a separate `types.ts` file?  It's not like this file is too big but just wondering if you think it's worth it based on patterns of other components.

**Review by taylorvnoj — COMMENTED** [review]
Quick contemplation!

File: packages/components-native/src/ThumbnailList/ThumbnailList.tsx (hunk 1 above)
Comment (MichaelParadis) [reply packages/components-native/src/ThumbnailList/ThumbnailList.tsx:13]: Makes sense to me. I can update it

**Review by taylorvnoj — APPROVED** [review]
Thanks for making that change!

**Pull request merged**